config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
```

Comment:
I don't fully understand what's wrong. Should I delete the file or just the first 3 lines?